dmcontent/govuk_frontend.py

```diff
@@ -377,10 +377,20 @@ def dm_multiquestion(
 
             for item in items:
                 if item["value"] in followups:
-                    followup_q = question.get_question(followups[item["value"]][0])
-                    item["conditional"] = {
-                        "html": from_question(followup_q, data, errors, is_page_heading=False)
-                    }
+                    if len(followups[item["value"]]) > 1:
+                        number_of_followups = len(followups[item["value"]])
+                        followup_items = []
+                        for i in range(number_of_followups):
+                            followup_q = question.get_question(followups[item["value"]][i])
+                            followup_items.append(from_question(followup_q, data, errors, is_page_heading=False))
```

Comment:
In Python `for` loops can iterate over the elements in a list without needing to use `range`

```suggestion
                        followup_items = []
                        for followup_id in followups[item["value"]]:
                            followup_q = question.get_question(followup_id)
                            followup_items.append(from_question(followup_q, data, errors, is_page_heading=False))
```